orchagent/port.h

```diff
@@ -134,6 +141,7 @@ class Port
     std::vector<sai_object_id_t> m_priority_group_ids;
     sai_port_priority_flow_control_mode_t m_pfc_asym = SAI_PORT_PRIORITY_FLOW_CONTROL_MODE_COMBINED;
```

Comment:
We have two variable for save _port priority flow control mode_: `m_pfc_asym` and  `pfc_mode`  in  `struct PfcInfo`
 Maybe we can remove `m_pfc_asym` ?